botocore/data/appstream/2016-12-01/waiters-2.json

```diff
@@ -0,0 +1,55 @@
+{
+  "version": 2,
+  "waiters": {
+    "FleetStarted": {
```

Comment:
Are these correct? Looking at the [docs](https://docs.aws.amazon.com/appstream2/latest/APIReference/API_Fleet.html#AppStream2-Type-Fleet-State). It looks like the states should be `RUNNING` and `STOPPED`. There are also no `PENDING_*` states.

Reply:
Yeah it's correct.